Webpack "Cannot find module 'babel-plugin-redom-jsx'"

I'm setting up a new environment with redom and this "jsx" transformation with webpack 4 and babel 7. When I run webpack, though, I get this error:

    ERROR in ./src/index.js
    Module build failed (from ./node_modules/babel-loader/lib/index.js):
    Error: Cannot find module 'babel-plugin-redom-jsx'

My `.babelrc` is modified to follow `README.md`:

```
{
    "presets": ["@babel/preset-env"],
    "plugins": [
        "babel-plugin-redom-jsx",
        ["transform-react-jsx", {
            "pragma": "el"
        }]
    ]
}
```

Note that with only the preset in my `.babelrc` webpack runs fine.

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ npm install babel-plugin-transform-redom-jsx --save-dev
 ``` json
 {
     "plugins": [
-        "babel-plugin-redom-jsx",
+        "babel-plugin-transform-redom-jsx",
         ["transform-react-jsx", {
             "pragma": "el"
         }]

